### PR TITLE
Release v0.5.0

### PR DESCRIPTION
**Pull request by RamLavi**
**What this PR does / why we need it**:

**Special notes for your reviewer**:

```release-note
Release v0.5.0
```

### CODE REVIEW

File: version/description [hunk 1]
```diff
@@ -1,11 +1,7 @@
...
 Features:
-* Bump device-plugin-manager and kubelet deps (#30)
-* Bump dependencies (#29)
-Bugs:
-
+* tier1, Change tests to get namespace from cluster
+* tier1, Change KUBECONFIG to be configurable in functest
 ```
-docker pull quay.io/kubevirt/macvtap-cni:v0.4.0
+docker pull HANDLER_IMAGE
```
Comment (phoracek) [inline version/description:6]: AFAIK there is a bug/missing feature in release scripts - we never replace this placeholder. Please put in the image name manually.

Comment (RamLavi) [reply version/description]: right. 
[Done](https://github.com/kubevirt/macvtap-cni/compare/684314a4bcc23a7a47b5b56e80d5f714fa5bbe90..cf08f54ff1b38b277c9f3a609f776f1c5a21045f)

**Review by maiqueb — CHANGES_REQUESTED** [review]
I'm not comfortable bumping a Y stream because of something in the tests. Is there a reason why we can't bump the Z ?

File: version/description [hunk 2]
```diff
@@ -1,11 +1,7 @@
...
-Bump dependencies to align with kubernetes v1.19.1
-
+v0.5.0
 Features:
-* Bump device-plugin-manager and kubelet deps (#30)
-* Bump dependencies (#29)
-Bugs:
-
+* tier1, Change tests to get namespace from cluster
+* tier1, Change KUBECONFIG to be configurable in functest
```
Comment (maiqueb) [inline version/description:4]: These are the commits; none of this is a feature.

Write something like 'enable macvtap to be tested as part of CNAO tier1' .

**Comment by RamLavi** [comment]
> I'm not comfortable bumping a Y stream because of something in the tests. Is there a reason why we can't bump the Z ?

yeah it's kind of a grey area, Semver says that minor is for functionality changes, patch on bug fixes..
this is more of an added functionality IMO, but if you prefer I will issue this as patch.. 
your call :)

**Comment by maiqueb** [comment]
> > I'm not comfortable bumping a Y stream because of something in the tests. Is there a reason why we can't bump the Z ?
> 
> yeah it's kind of a grey area, Semver says that minor is for functionality changes, patch on bug fixes..
> this is more of an added functionality than a bug IMO, but if you prefer I will issue this as patch..
> your call :)

Hm... Added functionality ... somewhere else. Macvtap-cni repo does exactly the same thing it did before, imho.

**Comment by RamLavi** [comment]
closing PR in favor of opening a patch release

**Pull request closed without merging**